DFE-Digital/schools-experience
# Add event driven SMS

### Trello card
https://trello.com/c/Vt2y63Sd

### Context
We want to alert the candidates by SMS when their placement request has been accepted, or it has been accepted then cancelled.

### Changes proposed in this pull request
- Remove unused SMS templates - Left these in by accident. They were used for testing in the previous PR
- Notify the candidate by SMS when the school cancels
- Notify the candidate by SMS when the school accepts a placement request
- Update the help text above the phone number input
- Update the confirmation screens to let the school know a text has been sent along with the email.

### Guidance to review
To test need to:
1. Be running the app locally, the preview app doesn't use Notify.
2. Create a new placement request.
3. Log in as the school
4. Accept the request. SMS should be sent
5. Visit the Manage upcoming bookings section and cancel the request. SMS should be sent.

To do: 
1. ~~Improve the help text above the phone number input?~~ ✅ 
Before:
![image](https://user-images.githubusercontent.com/47089130/145438490-392a3f56-7e76-4b6e-9764-d0e63e65bd21.png)
After:
![image](https://user-images.githubusercontent.com/47089130/145807942-305a80ad-01d0-42d2-8c6d-9a36ee25570c.png)

2. ~~Add some text to explain to the schools that they are also sending a text? These are sent when the school clicks `Send confirmation email` in both cases. There is a confirmation page per email sent too (I haven't actually updated the preview or confirmation screens yet, just included screenshots below for clarity).~~ ✅ 

| Preview | Confirmation | SMS |
| ------- | --------- | --------- | 
| ![image](https://user-images.githubusercontent.com/47089130/145436523-da112169-25ae-42dd-840b-a835657e44bf.png) | ![image](https://user-images.githubusercontent.com/47089130/145436654-2a954598-978f-4a93-b8c5-bcbc692d323d.png) | ![image](https://user-images.githubusercontent.com/47089130/145438790-44f4b06d-74d2-4f48-a9b9-99b7078c0e74.png) | 
![image](https://user-images.githubusercontent.com/47089130/145437300-25da8554-d116-43ff-a86f-bd2a6d05549a.png) | ![image](https://user-images.githubusercontent.com/47089130/145437426-03ef1250-e66c-4625-ab6d-acfe0bc574a8.png) | ![image](https://user-images.githubusercontent.com/47089130/145438233-c39121ae-bb1e-4ad9-96f2-221d10f1db0e.png) |

After: 
![image](https://user-images.githubusercontent.com/47089130/145808775-690c7a7b-8cd7-45bd-806d-3b3a0e18a617.png)
![image](https://user-images.githubusercontent.com/47089130/145808898-66bd3573-83ed-4afe-ab01-285f0f220d92.png)
![image](https://user-images.githubusercontent.com/47089130/145809976-c9c934df-696f-422d-984d-2e36f1c9d296.png)
